projects/manager/src/app/datasets/loading-info/gdal-metadata-list/gdal-metadata-list.component.html

```diff
@@ -0,0 +1,82 @@
+
+        <mat-divider></mat-divider>
+    </div>
+
+    <div formGroupName="rasterResultDescriptor">
+        <mat-form-field appearance="fill">
+            <mat-label>Output Data Type</mat-label>
+            <mat-select formControlName="dataType">
+                <mat-option *ngFor="let dataType of RasterDataTypes" [value]="dataType">
+                    {{ dataType }}
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+
+        <mat-form-field appearance="fill">
+            <mat-label>Spatial Reference</mat-label>
+            <input matInput type="text" formControlName="spatialReference" />
+        </mat-form-field>
+
+        <mat-form-field appearance="fill">
+            <mat-label>Band Name</mat-label>
+            <input matInput type="text" formControlName="bandName" />
+        </mat-form-field>
+    </div>
+
+    <div class="container">
+        <mat-nav-list class="left">
+            <mat-list-item
+                *ngFor="let control of form.controls.timeSlices.controls; let i = index"
+                (click)="selectTimeSlice(i)"
+                [class.selected]="selectedTimeSlice === i"
+            >
+                <h4 matListItemTitle>
+                    {{ getTime(i) }}
+                </h4>
+                <p matListItemLine>{{ control.controls.gdalParameters.value.filePath }}</p>
+            </mat-list-item>
+            <mat-error *ngIf="form.controls.timeSlices.errors?.overlappingTimeInterval">Time slices must not overlap.</mat-error>
+            <div class="empty-placeholder" *ngIf="form.controls.timeSlices.controls.length === 0">
+                <i>No time slices defined.</i>
+            </div>
+            <button class="add-entry" mat-raised-button (click)="addTimeSlicePlaceholder()">Add Time Slice</button>
+            <button *ngIf="form.controls.timeSlices.controls.length > 0" class="add-entry" mat-raised-button (click)="removeTimeSlice()">
+                Remove selected time slice
+            </button>
+        </mat-nav-list>
+
+        @if (form.controls.timeSlices.controls.at(selectedTimeSlice); as selectedTimeSliceGroup) {
+            <div class="time-slice" [formGroup]="selectedTimeSliceGroup">
+                <geoengine-time-interval-input [formControl]="selectedTimeSliceGroup.controls.time"></geoengine-time-interval-input>
+
+                <mat-divider></mat-divider>
+
+                <geoengine-manager-gdal-dataset-parameters
+                    [form]="selectedTimeSliceGroup.controls.gdalParameters"
+                    [dataPath]="dataPath"
+                ></geoengine-manager-gdal-dataset-parameters>
+
+                <mat-divider id="gdal-params-divider"></mat-divider>
+
+                <div class="cachettl">
+                    <mat-form-field appearance="fill">
+                        <mat-label>CacheTTl</mat-label>
+                        <input matInput type="number" [formControl]="selectedTimeSliceGroup.controls.cacheTtl" />
```

Comment:
i guess we need some information and/or good defaults here...

Reply:
I guess 0 (= don't cache) is a good default. I tried to explain it a bit better: 64634ca05a47024ee2de403e96a4285354ea044d

Reply:
well then we will not use the cache so the default should be at least one day. Maybe inf is also fine.  The single file on disk will probably never change.

Reply:
yes we could do some kind of versioning. Maybe we can postpone this and merge this soon?

Reply:
21f877da47c4110588e8fe2252081ff6da190c63